README.md

```diff
@@ -81,6 +81,7 @@ We use the div to control the entire height and width, and the span to get acces
 * `columns`: an array of columns to create, on order. Each entry is an object with the following parameters:
 	* `tree`: boolean, whether the jstree should be placed in this column. Only the first `true` is accepted. If no column is set to `tree:true`, then the first column is used.
 	* `width`: width of the column in pixels. If no width is given, the default is `auto` **except for the last column**. In the last column, if no width is given, it is treated as 'auto' and fills the entire rest of the grid to the right.
+	* `ellipsis`: true/false. If true (the default), then, if the column content overflows the column width, an ellipsis (…) is shown at the end.
```

Comment:
Based on the commit that vakata did, I was thinking:

* `true`: always show an ellipsis if too long (i.e. apply the class)
* `false`: never show an ellipsis if too long (current default)
* `undefined` (in other words, user does not pick anything): follow whatever the rule is for the tree itself.

Thoughts?